chapters/introduction.tex

```diff
@@ -51,6 +51,7 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 
 Therefore, a flat hybrid DAE is seen as a set of equations where some of the equations are only conditionally evaluated.
 Initial setup of the model is specified using \lstinline!start!-attributes and equations that hold only during initialization.
+In terms of terminology the simulation starts with solving the initialization problem at the starting time, and then simulates forward in time from those computed initial values.
```

Comment:
I'd prefer making proper definitions in the next section, so that they appear in the index, and that we don't put what is actually definitions right above the section named _Some Definitions_.

Reply:
Done!

Reply:
To be continued here: https://github.com/modelica/ModelicaSpecification/pull/3146#discussion_r843502313

Resolving.